rhobs/alerting/data_plane/prometheus.latency_pac_provision_alerts.yaml

```diff
@@ -19,7 +19,7 @@ spec:
           ) / ignoring(le)
           increase(redhat_appstudio_buildservice_PaC_configuration_time_bucket{le="+Inf"}[5m])
         ) > 0.05
-      for: 1m
+      for: 10m
```

Comment:
I don't think this is an alert owned by the integration service, if we want to update it, we should probably notify the build service team about it.